### DISCUSSION TITLE
Mods not working on Android

### DISCUSSION THREAD

**Discussion by DropzoneJD** [exult/exult#555] [General]
Hoping someone can help me out here. Been playing U7 BG on my Galaxy Fold and I can't seem to get the mods to work. I've downloaded the zip files, installed as per the exult instructions (using the check marks in the UI). They seem to install just fine, but when I launch the game they aren't there.

Looking at the files on my phone, the mod folders and files appear to be in the correct place. 

Any suggestions?

**Comment by DominusExult** [comment]
I‘ll take a look. So far only our mods BG Keyring and SI Fixes work directly, all others need a workaround (basically using our mods as trojan horses ;)). But I haven‘t tried that myself yet.

**Reply by DropzoneJD** [reply]
Thank you, I appreciate it! Those are the only mods I'm trying to use.

**Comment by DominusExult** [comment]
I've opened an issue for this #558 - I can reproduce the problem and it seems to me as if the Exult launcher hangs at unzipping the files.

**Reply by DropzoneJD** [reply]
I tried manually extracting them to the appropriate folder but that's not working either. My device isn't rooted though and it's possible I've got the location wrong. Not sure if this changes anything. Thanks again for looking into this. I've got a long flight to Japan coming up and I plan to kill my time playing my all time favorite game!

**Reply by DominusExult** [reply]
I found that the last working version is https://github.com/exult/exult/releases/tag/snapshot-v1.9.0.20240502 - So best get this before your flight and enjoy playing :)
In the meantime I'll try to find out what went wrong.

**Reply by DropzoneJD** [reply]
Thank you! What an awesome community!
